metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala

```diff
@@ -526,19 +424,68 @@ private class BloopPants(
       ),
       java = Some(C.Java(Nil)),
       sbt = None,
-      test = test,
+      test = bloopTestFrameworks,
       platform = Some(C.Platform.Jvm(C.JvmConfig(javaHome, Nil), None)),
-      resolution = resolution
+      resolution = None
     )
   }
 
+  val myExportClasspath: mutable.Map[String, List[Path]] =
+    mutable.Map.empty[String, List[Path]]
+  val exportClasspathDir: AbsolutePath = AbsolutePath(
+    workspace.resolve("dist").resolve("export-classpath")
+  )
+  def exportClasspath(target: PantsTarget): List[Path] = {
+    myExportClasspath.getOrElseUpdate(
+      target.name, {
+        val classpathFile =
+          exportClasspathDir.resolve(target.id + "-classpath.txt")
+        if (classpathFile.isFile) {
+          Classpath(classpathFile.readText.trim()).entries.map(_.toNIO)
+        } else {
+          Nil
+        }
+      }
+    )
+  }
   lazy val dist: mutable.Buffer[Path] = Files
     .list(
       workspace.resolve("dist").resolve("export-classpath")
     )
     .asScala
     .toBuffer
 
+  def bloopTestFrameworks: Option[C.Test] = {
+    val scalatest = C.TestFramework(
+      List(
+        "org.scalatest.tools.Framework",
+        "org.scalatest.tools.ScalaTestFramework"
+      )
+    )
+    // These test frameworks are the default output from running `show
+    // testFrameworks` in sbt (excluding spec2). The output from `./pants
```

Comment:
How come we are excluding spec2?

Reply:
Updated the list to be exactly the same as default output from sbt.